rest/model/patch.go

```diff
@@ -97,8 +99,9 @@ func (p *APIParameter) ToService() patch.Parameter {
 	return res
 }
 
-// BuildFromService converts from service level structs to an APIPatch
-func (apiPatch *APIPatch) BuildFromService(p patch.Patch) error {
+// BuildFromService converts from service level structs to an APIPatch.
+// If withArgs is set, includes identifier and commit queue position from the DB, if applicable.
+func (apiPatch *APIPatch) BuildFromService(p patch.Patch, withArgs bool) error {
```

Comment:
Since withArgs is seems to be used as a "isThisACommitQueuePatch" bool, do we maybe want to rename it to something more closely related to that?

Reply:
Well it does also include the identifier thing. I think it's also not for if it's a commit queue but rather if we want to check that it is.